base/templates/base.html

```diff
@@ -18,8 +18,7 @@
                     </form>
                 </div>
             {% else %}
-                <!-- TODO broken for some reason. should use url with reverse -->
```

Comment:
Any reason you switched from django template comment to an HTML comment? This one will actually show up in the page. Where a Django template comment will not.

Reply:
Oh, I didn't realize. I just used the shortcut key and it did it that way, so I made the others match.